athena/modules/text/module_text_llm/module_text_llm/chain_of_thought_approach/generate_suggestions.py

```
@@ -3,122 +3,154 @@
 from llm_core.utils.llm_utils import (
-    get_chat_prompt_with_formatting_instructions, 
-    check_prompt_length_and_omit_features_if_necessary, 
+    check_prompt_length_and_omit_features_if_necessary,
+    get_chat_prompt,
     num_tokens_from_prompt,
 )
-from llm_core.utils.predict_and_parse import predict_and_parse
-from module_text_llm.helpers.utils import add_sentence_numbers, get_index_range_from_line_range, format_grading_instructions
-from module_text_llm.chain_of_thought_approach.prompt_thinking import InitialAssessmentModel
-from module_text_llm.chain_of_thought_approach.prompt_generate_feedback import AssessmentModel
+from llm_core.core.predict_and_parse import predict_and_parse
```

Pylint (3.3.7)

[error] Unable to import 'llm_core.core.predict_and_parse'

(E0401)